expense/activities.py

```diff
@@ -0,0 +1,89 @@
+import httpx
+from temporalio import activity
+
+from expense import EXPENSE_SERVER_HOST_PORT
+
+
+@activity.defn
+async def create_expense_activity(expense_id: str) -> None:
+    if not expense_id:
+        raise ValueError("expense id is empty")
+
+    async with httpx.AsyncClient() as client:
```

Comment:
Consider making the activities as methods on a class that instantiates this in `__init__` and reuses it. It is a bit better performance to reuse an HTTP client/connection and I think a better practice to demonstrate to users